wpa_supplicant/ctrl_iface_zephyr.c

```diff
@@ -120,25 +120,28 @@ static void wpa_supplicant_ctrl_iface_receive(int sock, void *eloop_ctx,
 					      void *sock_ctx)
 {
 	struct wpa_supplicant *wpa_s = eloop_ctx;
-	char *buf, *pos;
+	char buf[CTRL_IFACE_MAX_LEN + 1];
```

Comment:
should this buffer be initialized to zero?

Reply:
No need, it's processed for every event explicitly